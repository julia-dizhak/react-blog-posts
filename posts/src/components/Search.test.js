import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer'; 
import SearchForm from './Search';

describe('Search form', () => {
    it.skip('renders without crashing', () => {
        const div = document.createElement('div');
        ReactDOM.render(<SearchForm>Search</SearchForm>, div);
        ReactDOM.unmountComponentAtNode(div);
    });
    
    test.skip('has a valid snapshot', () => {
        const component = renderer.create(
        <SearchForm>search</SearchForm>
    );
    
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});