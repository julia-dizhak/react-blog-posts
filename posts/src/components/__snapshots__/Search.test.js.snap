// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search form has a valid snapshot 1`] = `
<form
  className="search"
  onSubmit={undefined}
>
  search
  <input
    onChange={undefined}
    type="search"
    value={undefined}
  />
  <button
    className="button-search"
    onSubmit={undefined}
    type="submit"
  >
    Search
  </button>
</form>
`;
